client/src/components/ratings_reviews/RatingsAndReviews.js

```diff
@@ -0,0 +1,64 @@
+import React, {component, useState, useEffect} from 'react';
+import ReviewsList from './ReviewsList.js';
+import exampleData from '../../store/exampleData.js';
+import axios from 'axios';
+import GITHUB_API_KEY from '../../config/config.js';
+import NewReview from './NewReview.js';
+import SideBar from './SideBar.js';
+
+// in this component i want to fetch all the reviews and send the relevent information to the review list component
+
+const RatingsAndReviews = () => {
+  // these are the states for the url and review components
+  const [reviewData, setReviewData] = useState([]);
+  const [pageNumber, setPageNumber] = useState(1);
+  const [count, setCount] = useState(2);
+  const [sort, setSort] = useState('relevant');
+  const url = 'https://app-hrsei-api.herokuapp.com/api/fec2/hratx/reviews/';
+  // states are needed for the sidebar component
+  const [metaData, setMetaData] = useState({});
```

Comment:
If we are using Redux, we would not use the useState() hook. We would pass the props in on line 11, via the useDispatch and useSelector hooks available in Redux:  https://thoughtbot.com/blog/using-redux-with-react-hooks